File: backend/local-api-module/src/main/java/org/vyatsu/localApiModule/dto/response/api/TokenDto.java
package org.vyatsu.localApiModule.dto.response.api;

import lombok.Value;
import org.vyatsu.localApiModule.entity.enums.TokenType;

import java.io.Serializable;

/**
 * DTO for {@link org.vyatsu.localApiModule.entity.user.Token}
 */
@Value
public class TokenDto implements Serializable {
    Long id;
    String token;
    TokenType type;
    boolean revoked;
    boolean expired;
}

File: app/src/main/java/com/bytebusters/android/mpsystem/dto/response/api/TokenDto.kt
package com.bytebusters.android.mpsystem.dto.response.api

import com.bytebusters.android.mpsystem.entity.enums.TokenType

data class TokenDto (
    var id: Long? = null,
    var token: String? = null,
    var type: TokenType? = null,
    var revoked: Boolean = false,
    var expired: Boolean = false
)